typescript/sdk/src/deploy/verify/ContractVerifier.ts

```diff
@@ -183,32 +191,39 @@ export class ContractVerifier {
       throw new Error(`[${chain}] ${errorMessage}`);
     }
 
+    verificationLogger.trace(
+      { apiUrl, chain, result: responseJson.result },
+      'Returning result from explorer.',
+    );
     return responseJson.result;
   }
 
-  private async isAlreadyVerified(
-    chain: ChainName,
-    input: ContractVerificationInput,
-    verificationLogger: Logger,
-  ): Promise<boolean> {
-    try {
-      const result = await this.submitForm(
-        chain,
-        ExplorerApiActions.GETSOURCECODE,
-        verificationLogger,
-        {
-          address: input.address,
-        },
-      );
-      return !!result[0]?.SourceCode;
-    } catch (error) {
-      verificationLogger.debug(
-        { error },
-        'Error checking if contract is already verified',
-      );
-      return false;
-    }
-  }
+  // private async isAlreadyVerified(
```

Comment:
what is your goal in commenting this out? do you expect to use this again in the future?

Reply:
I expect us to reward the function in the future, but also commented code is dead code. I'll remove it and we can come back to this PR if we really need that code.